docker-images/kafka/Makefile

```diff
@@ -31,4 +32,7 @@ docker_build: .kafka-agent.tmp .tracing-agent.tmp .mirror-maker-agent.tmp .third
 
 include ../../Makefile.docker
 
+docker_build:
+	../find-classpath-collision.sh strimzi/$(PROJECT_NAME):$(BUILD_TAG) /opt/kafka/libs "tmp/$(KAFKA_VERSION).whitelist"
```

Comment:
I forgot how exactly does the Makefile stuff work ... will this not override the regular docker_build target?

Reply:
It should append it to the included recipe.